custom_manifests/README.md

```diff
@@ -1,22 +1,32 @@
-These manifest files are specifically designed for the Development Preview of OpenShift 4.14 at Oracle Cloud Infrastructure (OCI). They are used in the 6th step: "Custom manifests", of provisioning an OpenShift cluster in RedHat's Hybrid Cloud Console / OpenShift UX.
-
-`cloud-provider.yaml` in `oci-ccm.yml` and `config.yaml` in `oci-csi.yml` should be replaced by the output of the OCI Resource Manager Stack (RMS) job.
-
-example:
-```
-stringData:
-  cloud-provider.yaml: |
-    useInstancePrincipals: true
-    compartment: <compartment-ocid>
-    vcn: <vcn-ocid>
-    loadBalancer:
-      subnet1: <subnet-ocid>
-      securityListManagementMode: Frontend
-      securityLists:
-        <subnet-ocid>: <security-ocid>
-    rateLimiter:
-      rateLimitQPSRead: 20.0
-      rateLimitBucketRead: 5
-      rateLimitQPSWrite: 20.0
-      rateLimitBucketWrite: 5
-```
+# Custom Manifests
+
+These OpenShift and Kubernetes manifest files support the installation of Red Hat OpenShift clusters on Oracle Cloud Infrastructure. The Butane files used to generate the OpenShift MachineConfigs are also included.
+
+View usage during [installation](/README.md#documentation-and-installation-instructions)
+
+
+### Individual Manifests
+---
+| File | Description | When to Use |
+--- | --- | ---
+**01-oci-ccm.yml** | Cluster resources for OCI Cloud Controller Manager (CCM). | Always ✅
+**01-oci-csi.yml** | Cluster resources for OCI Cloud Storage Interface (CSI). See [STORAGE.md](/STORAGE.md) | Always ✅
+**01-oci-driver-configs.yml** | Configuration Secrets for CCM and CSI drivers. ❗**Contains placeholder values that need to be replaced before use.** | Always ✅
+**02-machineconfig-ccm.yml** | MachineConfig that fetches the provider (OCI) id for kubelet from the OCI Metadata of the instance. | Always ✅
```

Comment:
Is "OCI Metadata" a resource or object that is always capitalized, or can we say "OCI metadata"?

Reply:
we should say "OCI metadata"